ddp-workspace/projects/toolkit/src/lib/components/dashboard/dashboard-redesigned.component.ts

```diff
@@ -51,37 +80,119 @@ import { map, take, filter } from 'rxjs/operators';
+                            <div class="dashboard-content dashboard-content_table">
+                                <ddp-user-activities [studyGuid]="studyGuid"
+                                                     [displayedColumns]="toolkitConfig.dashboardDisplayedColumns"
+                                                     [participantGuid]="participantGuid"
+                                                     [activities]="isParticipantOperator(participantGuid) ? operatorActivities : null"
+                                                     (open)="navigate($event, participantGuid)">
+                                </ddp-user-activities>
+                            </div>
+                        </ng-template>
                     </div>
                 </section>
             </ng-template>
-        </main>`
+        </main>`,
+    styles: [`
+        .full-width {
+            width: 100%;
+        }
+    `]
 })
 export class DashboardRedesignedComponent extends DashboardComponent implements OnInit {
     public invitationId: string | null = null;
+    public dashboardParticipants$: Observable<DashboardParticipant[]>;
+    public panelsState = new Map<string, boolean>();
+    public operatorActivities: Array<ActivityInstance>;
 
     constructor(
         private headerConfig: HeaderConfigurationService,
-        private session: SessionMementoService,
-        private _router: Router,
+        protected session: SessionMementoService,
+        protected router: Router,
         private _announcements: AnnouncementsServiceAgent,
         private userInvitation: UserInvitationServiceAgent,
         _participantsSearch: ParticipantsSearchServiceAgent,
-        @Inject('toolkit.toolkitConfig') public config: ToolkitConfigurationService) {
-        super(_router, _announcements, _participantsSearch, config);
+        private governedParticipantsAgent: GovernedParticipantsServiceAgent,
+        private userActivityServiceAgent: UserActivityServiceAgent,
+        private userProfile: UserProfileServiceAgent,
+        private translate: TranslateService,
+        @Inject('ddp.config') private config: ConfigurationService,
+        @Inject('toolkit.toolkitConfig') public toolkitConfig: ToolkitConfigurationService) {
+        super(router, _announcements, _participantsSearch, session, toolkitConfig);
     }
 
     public ngOnInit(): void {
         super.ngOnInit();
         this.headerConfig.setupDefaultHeader();
         !this.isAdmin && this.getInvitationId();
+
+        this.dashboardParticipants$ = this.getDashboardParticipants();
+    }
+
+    private getOperatorActivities(): Observable<Array<ActivityInstance> | null> {
+        return of(this.session.session.userGuid).pipe(
+            tap((userGuid) => {
+                this.session.setParticipant(userGuid);
+            }),
+            switchMap(() => this.userActivityServiceAgent.getActivities(of(this.config.studyGuid))));
+    }
```

Comment:
the thing is `setParticipant` should be done exactly in the data stream, at emission. Not when we create the stream. Otherwise we can set multiple participants and only after that call for activities with the last set participant. I updated the code and added a comment about that.
Also about you suggestion to move setParticipant into getActivities method. I thought it over, even implemented it, but it looked less obvious then why getActivities service can't be called before subscription happened, we have to add a comment about that, which is not making the situation better.